src/components/Home/CareerPart2.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Data from "./JobsData.json";
+import "../styles/careersection.css";
+
+const CareerPart2 = () => {
+    return(
+        <div className="div">
+            <h1 className="header">current open positions</h1>
+            <h3 className="h3">Please send us an email with the application title as the subject with an attached CV in PDF format!</h3>
+            <table className="sm:justify-center ml-[2%]">
+           {
+            
+            Data && Data.map( record => {
+                return(
+                    <div className="sm:justify-center">

```

Comment:
I don't think a table should contain a div block, in order to wrap your tr and td, you can use React fragment which will wrap your elements without inserting any block.

You can implement it in the following 2 ways: 
<React.Fragment>
"insert your code here"
</ React.Fragment>

or
<>
"Insert your code here"
</>

And the key={record.id} should be inside the react fragment

For more info, https://reactjs.org/docs/fragments.html